packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/container.tsx

```diff
@@ -89,6 +105,20 @@ const ConfirmResetPasswordContainer = (_: RouteComponentProps) => {
     }
   };
 
+  const checkForTotp = async (token: string) => {
+    try {
+      return await authClient.checkTotpTokenExistsWithPasswordForgotToken(
+        token
+      );
+    } catch (error) {
+      console.log('Error checking for TOTP', error);
+      return {
+        exists: false,
```

Comment:
If users actually have TOTP but this failed for some reason, would it cause issues on the next page?